provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -0,0 +1,92 @@
+import os.path as P
+import subprocess as S
+from string import Template
+from typing import Dict, Optional, List
+
+import pkg_resources
+
+from hare_mp.store import ValueProvider, as_json, as_list
```

Comment:
How about s/hare_mp/hare?

Reply:
We already have hare component. Hare is bigger than this package. So I decided to differentiate them.